research/deep_speech/deep_speech.py

```diff
@@ -17,6 +17,7 @@
 from __future__ import division
 from __future__ import print_function
 
+
```

Comment:
Could you add a random seed to the main loop?

Reply:
Discussed offline, we don't need setting random seeds as it's behavior is not fully expected.